pkg/apis/networking/v1/defaults.go

```diff
@@ -43,3 +45,15 @@ func SetDefaults_NetworkPolicy(obj *networkingv1.NetworkPolicy) {
 		}
 	}
 }
+
+func SetDefaults_Ingress(obj *networkingv1.Ingress) {
```

Comment:
cc @liggitt this should fix https://github.com/kubernetes/kubernetes/pull/92312#discussion_r521066786

Reply:
I did not realize this struct was reused for Ingress!  I don't think this is right.  It happens to work for the fields that exist today, but I don't know of *any* Ingress implementation that is VIP-like.

I think we should probably clone the definition of `type LoadBalancerStatus` into the networking apigroup.

@bowei @rramkumar1 @aledbf - agree or disagree?

Reply:
And if we do that, it should be a standalone prefactoring commit. :)

Reply:
> I think we should probably clone the definition of type LoadBalancerStatus into the networking apigroup.

+1

Reply:
Wouldn't that be a breaking change in client-go? 🤔